tools/StaticAnalysis/Exceptions/SignatureIssues.csv

```diff
@@ -1542,8 +1542,95 @@
 "Microsoft.Azure.Commands.Batch.dll","Microsoft.Azure.Commands.Batch.StartBatchPoolResizeCommand","Start-AzureBatchPoolResize","1","8410","Parameter TargetDedicatedComputeNodes of cmdlet Start-AzureBatchPoolResize does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.Commands.Batch.dll","Microsoft.Azure.Commands.Batch.StartBatchPoolResizeCommand","Start-AzureBatchPoolResize","1","8410","Parameter TargetLowPriorityComputeNodes of cmdlet Start-AzureBatchPoolResize does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.Commands.Batch.dll","Microsoft.Azure.Commands.Batch.GetBatchTaskCountsCommand","Get-AzureBatchTaskCounts","1","8400","Get-AzureBatchTaskCounts uses the noun 'AzureBatchTaskCounts', which does not follow the enforced naming convention of using a singular noun for a cmdlet name.","Consider using a singular noun for the cmdlet name."
+"Microsoft.Azure.Commands.ApplicationInsights.dll","Microsoft.Azure.Commands.ApplicationInsights.RemoveApplicationComponentInsightsApiKey","Remove-AzureRmApplicationInsightsApiKey","2","8210","Remove-AzureRmApplicationInsightsApiKey does not have a Force parameter but the cmdlet verb 'Remove' indicates that it may perform destructive actions under certain circumstances. Consider whether the cmdlet should have a Force parameter anduse ShouldContinue under some circumstances. ","Consider wether the cmdlet should have a Force parameter and use ShouldContinue under some circumstances. "
```

Comment:
Why did you add all of this to SignatureIssues? You need to address the 1 level issues and you can ignore level 2.

Reply:
reverted to origin